test/specs/events/edit.js

```diff
@@ -1,31 +1,27 @@
-var MainPage = require('../../pages/page');
-var EditEventPage = require('../../pages/events/edit');
-var MyDojosPage = require('../../pages/myDojos');
-var ManageEventsPage = require('../../pages/events/manage');
+// var MainPage = require('../../pages/page');
```

Comment:
Whoops, that entire file should be removed from the commit. It was just a quick spec for testing my page object, didn't actually test anything. create.js is the only new spec that should be added.